### PR TITLE
Clean up default alert rules

### PR DESCRIPTION
**Pull request by murrant**
Too many rules in the default, disabling the customoid rules.
It is easy to add from the collection.  Any other suggestions?

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: misc/alert_rules.json [hunk 1]
```diff
@@ -438,24 +438,24 @@
  {
     "rule": "customoids.customoid_current >= `customoids.customoid_limit` && customoids.customoid_alert = \"1\" && macros.device_up = \"1\"",
     "name": "CustomOID over limit",
-    "default": true
```
Comment (Jellyfrog) [inline misc/alert_rules.json:441]: Isn't it enough to simply remove the key completely to disable it?

**Comment by Jellyfrog** [comment]
I would suggest removing bgp ones also in that case. Bit too advanced/specific for most users

File: misc/alert_rules.json (hunk 1 above)
Comment (murrant) [reply misc/alert_rules.json:441]: sure

**Comment by SourceDoctor** [comment]
i'm with jellyfrog,
but it would be a shame to just throw them away, maybe doing something like alert rule packages?
Often thought about it, especially for applications.

**Review by Jellyfrog — APPROVED** [review]

**Comment by murrant** [comment]
@SourceDoctor all rules in the is file are in the "collection"

**Comment by SourceDoctor** [comment]
it seems this PR crashes some tests on Travis

**Comment by murrant** [comment]
Oops didn't test after removing the lines instead of changing them to false...

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-67-release-changelog-august-2020/13285/1

**Pull request merged**